src/epitools/widgets/cell_statistics.py

```diff
@@ -88,7 +88,7 @@ def _create_colour_labels_widgets() -> list[Widget]:
         label="statistic",
         widget_type="ComboBox",
         options={
-            "choices": ["id", "area", "perimeter", "orientation", "neighbours"],
+            "choices": ["id", "orientation", "neighbours"],
```

Comment:
These only work with the unit conversions, but I would need to work them out for all other `regionprops` so just removing the colourmaps for now.